microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -190,16 +190,29 @@ You can access Collections from an InPrivate window when using a signed-out prof
 <!-- ====================================================================== -->
 ## Compose
 
+#### Draft with Compose in Edge Sidebar
+
 In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the webpage directly.
 
 ![Compose UI](./index-images/compose.png)
 
 To access Compose, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of Microsoft Edge, and then select **Compose**. Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM). 
 
-Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.
-
 To change settings for the sidebar or Copilot pane, go to `edge://settings/sidebar`, and then select **App and notification settings** > **Copilot**.
 
+#### Rewrite on any webpage with Compose 
+Compose can also help rephrase what you've written and iterate using the same **Tone**, **Format**, and **Length** selections. 
+
+To access Compose directly in a webpage, highlight text longer than 5 words in any input text field. Click the "Rewrite with Copilot" button or Alt+I shortcut to start .  Compose may not be available on some webpages and certain input fields.
+
+
+![Compose UI](./index-images/compose-inline.png)
+
+To disable Compose on all webpages, go to `edge://settings/languages`, and then toggle off **Collaborate with Copilot as you write on the web**. 
+
+Text you've highlighted or inputted are sent to Microsoft only after "Rewrite" or "Generate" buttons are clicked, not while selections are being made. No additional webpage data is ever shared. Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.
```

Comment:
Do we really send "IDs that are associated with your Microsoft Edge profile and device"?

Reply:
I believe AnaheimIDs and DeviceIDs (?) are sent with each query as per our privacy whitepaper

Reply:
Just confirmed both are sent in [whitepaper](https://microsoft.sharepoint.com/:w:/t/Edge/ERovfJIhynVBmkpCq1Pba_EBSeTPoNAMq9xIceElLIiKPA?e=ryRWvd)